### DISCUSSION TITLE
Mão na massa - Editor HTML

### DISCUSSION THREAD

**Discussion by richallyson** [balta-io/2801#64] [Perguntas e Respostas] [answered]
Bem, tava suave fazendo tudo, porém, cheguei em um ponto do programa que eu vi que futuramente ai dar algo de errado. Mas prossegui (lembrando que to usando o .net5, pois não tinha dado nenhum problema, até agora). Pois bem, estava criando a regular expression pra poder fazer a pesquisa de strings que ficam entre os strongs, e quando criei ela, ficou assim: 
![image](https://user-images.githubusercontent.com/24733986/163501340-39d66180-4b22-43b7-8aed-f824d125574f.png)

Ao invés de o parâmetro do regex ficar todo verdinho como uma string deve ser, o texto ficou laranja, e apenas os parênteses ficaram verdes. E por causa disso, aconteceu isso:
![image](https://user-images.githubusercontent.com/24733986/163501476-e81dedc8-0161-44d7-a501-3eee5c6d4f93.png)

Ele pega tudo depois do primeiro '>'. Queria saber se isso é algo especifico do .net5, se nessa versão o regex foi alterado. Eu tinha dado uma mudada no código, pra fazer umas coisas diferentes e treinar, mas logo após fiz o código exatamente como o da aula, mas esse "erro" persiste.

Desde já, brigadão pela resposta!

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @richallyson como vai?
Não houve mudanças no Regex, ele é um padrão global, não afeta apenas .NET...
Os regex são complexos mesmo, este artigo pode te ajudar:
https://www.geeksforgeeks.org/how-to-validate-html-tag-using-regular-expression/

**Reply by richallyson** [reply]
Brigadão, Balta!

**Comment by AleLucasG** [comment]
O meu código não reconhecê o caminho informado.
Erro: System.UnauthorizedAccessException: 'O acesso ao caminho 'C:\Usuarios' foi negado.'

**Reply by andrebaltieri** [reply]
Bom dia @AleLucasG, utilize outro caminho, cujo o Windows tenha acesso de execução!
